dts/bindings/cpu/arm,cortex-a55.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright 2022 NXP
```

Comment:
There is a race about this binding https://github.com/zephyrproject-rtos/zephyr/pull/52873/files#diff-c63bbaf10db596182beebe1b661bfc749ead826d42eaba2c6aab55c2d303e189

Reply:
Thanks, will remove the binding from this PR and add the PR introducing it as a dependency.